pkg/skeleton/index.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+<!-- /**
+ * @license
+ * Copyright (c) 2022 Google LLC All rights reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file.
+ */ -->
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no, viewport-fit=cover">
+<meta name="apple-mobile-web-app-capable" content="yes">
+
+<title>ArcsJs Skeleton</title>
+
+<link rel="icon" href="./conf/icon.png">
```

Comment:
it won't let me comment on the icon, and rightly so, because halloween is already in the past, but... shouldn't it be a real `skeleton` icon?!
like here, welcome, play with our software  ☠️

Reply:
ROFL